selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/DashboardWorkspace.java

```diff
@@ -99,7 +99,7 @@ public String getName() {
         "//div[contains(@class,'che-toolbar')]//span[contains(text(),'%s')]";
     String WORKSPACES_LIST = "//ng-transclude[@class='che-list-content']";
     String WORKSPACE_ITEM_NAME =
-        "//div[contains(@class, 'che-list-item-name')]/span[contains(text(),'%s')]";
+        "//div[contains(@class, 'workspace-name-clip')]//div[contains(@data-str, '%s')]";
```

Comment:
It may be reasonable to use debug-id here instead of xpath,

Reply:
It will be done by https://github.com/eclipse/che/issues/6686.